### PR TITLE
AHC into nexus repo

### PR DESCRIPTION
**Pull request by DarthHater**
Adding AHC into using UI chapter now that it's integrated.

### CODE REVIEW

**Comment by bentmann** [comment]
The screenshots are misguiding users by providing bad examples. The "A" in AHC stands for "Application", not component. The service is not meant to be used for on individual components like junit, it should be used with a collection of components that make up one's application. So a better example would be the Nexus distro or the Webgoat WAR.

File: chapter-using-nexus-ui.asciidoc [hunk 1]
```diff
@@ -386,6 +386,13 @@ To delete a component press the 'Delete component' button as shown in <<fig-comp
 up to confirm the deletion. You can only delete components from hosted and proxy repositories. A deletion of a
 components triggers the deletion of all its associated assets, in most repository formats.
 
+image::figs/web/analyze-application.png
+
+To analyze a component press the 'Analyze component' button as shown in <<fig-component-details>>. A modal will pop
```
Comment (bentmann) [inline chapter-using-nexus-ui.asciidoc]: The button is (intentionally) called "Analyze application", not "Analyze component".

File: chapter-using-nexus-ui.asciidoc [hunk 2]
```diff
@@ -386,6 +386,13 @@ To delete a component press the 'Delete component' button as shown in <<fig-comp
 up to confirm the deletion. You can only delete components from hosted and proxy repositories. A deletion of a
 components triggers the deletion of all its associated assets, in most repository formats.
 
+image::figs/web/analyze-application.png
+
+To analyze a component press the 'Analyze application' button as shown in <<fig-component-details>>. A modal will pop
```
Comment (bentmann) [inline chapter-using-nexus-ui.asciidoc]: `To analyze an "application" ...`

File: chapter-using-nexus-ui.asciidoc [hunk 3]
```diff
@@ -386,6 +386,13 @@ To delete a component press the 'Delete component' button as shown in <<fig-comp
...
+up to request further information from you such as email address, report password, a list of properitary packages
```
Comment (bentmann) [inline chapter-using-nexus-ui.asciidoc]: Typo in "proprietary"

File: chapter-using-nexus-ui.asciidoc [hunk 4]
```diff
@@ -386,6 +386,13 @@ To delete a component press the 'Delete component' button as shown in <<fig-comp
...
+for the component, and a name for the report. Once you have provided this information, press the Analyze button as
```
Comment (bentmann) [inline chapter-using-nexus-ui.asciidoc]: "the **application**"

Comment (bentmann) [inline chapter-using-nexus-ui.asciidoc]: `'Analyze' button`

File: chapter-using-nexus-ui.asciidoc (hunk 2 above)
Comment (bentmann) [inline chapter-using-nexus-ui.asciidoc]: Does the occasional reader now what "analyzing" an application entails?

"Is this going to send my private app to Sonatype"?

File: chapter-using-nexus-ui.asciidoc [hunk 5]
```diff
@@ -386,16 +386,19 @@ To delete a component press the 'Delete component' button as shown in <<fig-comp
...
 
+[[fig-analyze-application]]
+.Analyze Application Form
 image::figs/web/analyze-application.png
 
-To analyze an application press the 'Analyze application' button as shown in <<fig-component-details>>. A modal will pop
-up to request further information from you such as email address, report password, a list of proprietary packages
-for the application, and a name for the report. Once you have provided this information, press the Analyze button as 
-shown in <<fig-analyze-application>>. Your report link will be emailed to you as soon as it is finished.
+To analyze an application, press the 'Analyze application' button as shown in <<fig-component-details>>. A form
```
Comment (dulanism) [inline chapter-using-nexus-ui.asciidoc]: I often use @djsauble 's Taxonomy Guide when writing about features. So, as a little background, that's why I updated "modal" to form. Reference: https://docs.google.com/presentation/d/1fTVS-1NBuqIRJAth6PomkqKz2Nc5jCmzZ2MgtNpt7ZY/edit#slide=id.gb7601a8be_0_18

**Comment by dbradicich** [comment]
+1

File: chapter-using-nexus-ui.asciidoc [hunk 6]
```diff
@@ -386,9 +386,19 @@ To delete a component press the 'Delete component' button as shown in <<fig-comp
...
 
-NOTE: In some repository formats assets are shared across component and these remain after deletion a component
-deletion. For example, while a Docker image is a component and can be deleted, the layers that make it up remain
-after its deletion as these assets are potentially shared with other Docker images. 
+[[fig-analyze-application]]
+.Analyze Application Form
+image::figs/web/analyze-application.png
+
+To analyze an application, press the 'Analyze application' button as shown in <<fig-component-details>>. A form
+will pop up to request further information from you such as email address, report password, a list of
```
Comment (dbradicich) [inline chapter-using-nexus-ui.asciidoc]: 'such as' makes me think this is only a partial list, but its the whole list of fields, maybe simply ```will pop up to request further information from you: email address, report password, lahdy-dahdy-dah```

File: chapter-using-nexus-ui.asciidoc [hunk 7]
```diff
@@ -386,9 +386,19 @@ To delete a component press the 'Delete component' button as shown in <<fig-comp
...
+.Analyze Application Form
+image::figs/web/analyze-application.png
+
+To analyze an application, press the 'Analyze application' button as shown in <<fig-component-details>>. A form
+will pop up to request further information from you such as email address, report password, a list of 
+proprietary packages for the application, and a name for the report. Once you provide this information, 
+press the 'Analyze' button as shown in <<fig-analyze-application>>. Your report link will be emailed to you as 
+soon as it is finished.
+
+NOTE: In some repository formats assets are shared across components. They remain after a component deletion. For
```
Comment (dbradicich) [inline chapter-using-nexus-ui.asciidoc]: though a valid statement, what exactly does this have to do with AHC ?

Comment (DarthHater) [inline chapter-using-nexus-ui.asciidoc]: Previously followed the delete component blurb, I'll move it up.

**Comment by dulanism** [comment]
+1

**Comment by kellyrob99** [comment]
@dulanism Please hold off merging this until we schedule absorbing the code changes

**Comment by dulanism** [comment]
@kellyrob99 No prob

**Pull request merged**